daemon/server/router/image/image_routes.go

```diff
@@ -271,18 +273,18 @@ func (ir *imageRouter) postImagesLoad(ctx context.Context, w http.ResponseWriter
 		return err
 	}
 
-	var platform *ocispec.Platform
+	var platformList []ocispec.Platform
+	// platform param was introduce in API version 1.48
 	if versions.GreaterThanOrEqualTo(httputils.VersionFromContext(ctx), "1.48") {
-		if formPlatforms := r.Form["platform"]; len(formPlatforms) > 1 {
-			// TODO(thaJeztah): remove once we support multiple platforms: see https://github.com/moby/moby/issues/48759
-			return errdefs.InvalidParameter(errors.New("multiple platform parameters not supported"))
+		var err error
+		formPlatforms := r.Form["platform"]
+		// multi-platform params were introduced in API version 1.51
+		if versions.LessThan(httputils.VersionFromContext(ctx), "1.51") && len(formPlatforms) > 1 {
+			return errdefs.InvalidParameter(errors.New("multiple platform parameters are not supported in this API version; use API version 1.51 or later."))
```

Comment:
Ah, derp; this needs to be v1.52` now; https://github.com/moby/moby/blob/b0c22a931d65a56b10ba1a4e542c70ac1bd8ea1a/api/common.go#L5-L6